lib/fileset/default.nix

```diff
@@ -194,27 +211,40 @@ in {
 
     This variant is useful for tracing file sets in the Nix repl.
 
-    Type:
-      trace :: FileSet -> Any -> Any
-
-    Example:
-      trace (unions [ ./Makefile ./src ./tests/run.sh ]) null
-      =>
-      trace: /home/user/src/myProject
-      trace: - Makefile (regular)
-      trace: - src (all files in directory)
-      trace: - tests
-      trace:   - run.sh (regular)
-      null
-  */
-  trace =
-    /*
-    The file set to trace.
+
+    # Inputs
+
+    `fileset`
+
+    : The file set to trace.
 
     This argument can also be a path,
     which gets [implicitly coerced to a file set](#sec-fileset-path-coercion).
```

Comment:
This content is for the same argument. The last 2 lines should have two extra spaces of indentation so it's rendered correctly.